### PR TITLE
Changes to get weather tutorial working

### PR DESCRIPTION
**Pull request by KristinaRiemer**
These are up to the high resolution data section, which I'm still working on.

### CODE REVIEW

File: sensors/01-meteorological-data.Rmd [hunk 1]
```diff
@@ -68,31 +59,31 @@ The key is to process each type of met data (site, reanalysis, forecast, climate
...
-* /streams
-  * 
-* /datapoints
-  * &since=YYYY-MM-DD
-  * &until=YYYY-MM-DD
-  * stream_id=
+The data can be accessed using a URL. You can find data for a particular station, 
+a certain sensor from a station, or the datapoints from a sensor. Below are
+the URLs for the UA-MAC AZMET weather station, a certain sensor at that station, 
+and datapoints from that sensor for the month of January 2017.
```
Comment (Chris-Schnaufer) [inline sensors/01-meteorological-data.Rmd]: Maybe change this to be May 2018 to keep in line with the data we'll have available?

File: sensors/01-meteorological-data.Rmd [hunk 2]
```diff
@@ -68,31 +59,31 @@ The key is to process each type of met data (site, reanalysis, forecast, climate
...
+* Sensor: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/sensors/438/streams
+* Datapoints: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31
 
-1. find streams on "UA-MAC AZMET Weather Station" https://terraref.ncsa.illinois.edu/clowder/api/geostreams/sensors?sensor_name=UA-MAC AZMET Weather Station
-2. take sensor id from that response and find the streams: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/sensors/438/streams
-3. take stream id from that and use in the datapoints query: https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31
+All URLs have the same beginning (https://terraref.ncsa.illinois.edu/clowder/api/geostreams/), 
+then additional information is added for each type of data as shown below. 
 
+* Station: /sensors/sensor_name=[name]
```
Comment (Chris-Schnaufer) [inline sensors/01-meteorological-data.Rmd]: The example URL ends in a '/' so maybe get rid of that one above, or the leading ones here?

Comment (KristinaRiemer) [reply sensors/01-meteorological-data.Rmd]: See ae3aac94513e9b884a4113f6a3db8a6f8cae83d5

File: sensors/01-meteorological-data.Rmd (hunk 1 above)
Comment (KristinaRiemer) [reply sensors/01-meteorological-data.Rmd]: See c28bf31c8e165803838d90d909219e56e83f5e7a, though it looks like there's only weather data available for the month of April?

**Comment by KristinaRiemer** [comment]
@dlebauer I think I've addressed all of your comments with the most recent commits?

File: sensors/01-meteorological-data.Rmd [hunk 3]
```diff
@@ -144,88 +127,63 @@ The data represent 5 minute summaries aggregated from 1/s observations.
...
-ggplot(data = mac_weather) +
-  geom_point(aes(x = time, y = wind_speed), size = 0.1)
-```
-
-#### Rainfall
-
-```{r precipitation}
-ggplot(data = mac_weather) +
-  geom_point(aes(x = time, y = precipitation_rate), size = 0.1)
+ggplot(data = weather_data) +
```
Comment (dlebauer) [inline sensors/01-meteorological-data.Rmd]: please use a separate plot for each variable. For example: 

image

 see: https://rpubs.com/dlebauer/ciat-met

Comment (KristinaRiemer) [reply sensors/01-meteorological-data.Rmd]: Oh, when you said one plot I for some reason thought you meant both variables on the same plot. So you want one plot for just one of the variables?

Comment (KristinaRiemer) [reply sensors/01-meteorological-data.Rmd]: I thought you said you only wanted one plot, of either precipitation or wind speed?

**Comment by KristinaRiemer** [comment]
This can be merged once @dlebauer approves.

**Pull request merged**